### PR TITLE
Add upgrade instructions for Keptn 0.8.3

### PR DESCRIPTION
**Pull request by johannes-b**
Signed-off-by: Johannes <johannes.braeuer@dynatrace.com>

### CODE REVIEW

File: content/docs/0.8.x/operate/upgrade/index.md [hunk 1]
```diff
@@ -6,9 +6,92 @@ aliases:
...
+:warning: **Important:** Please update the Shipyard for each project that should have remediations enabled in a particular stage. The instructions are provided in *Step 2*. 
+
+### Step 1: Upgrade Keptn
+
+1. Before starting the update, we strongly advise to create a backup of your Keptn projects. To do so, please follow the instructions in the [backup guide](../../operate/backup_and_restore).
+
+1. To download and install the Keptn CLI for version 0.8.3, you can choose between:
+    * **Automatic installation of the Keptn CLI (Linux and Mac)**:
+
+      * The next command will download the 0.8.2 release from [GitHub](https://github.com/keptn/keptn/releases), unpack it, and move it to `/usr/local/bin/keptn`.
```
Comment (christian-kreuzberger-dtx) [inline content/docs/0.8.x/operate/upgrade/index.md:23]: Should be 0.8.3?

File: content/docs/0.8.x/operate/upgrade/index.md [hunk 2]
```diff
@@ -6,9 +6,92 @@ aliases:
...
+```console
+curl -sL https://get.keptn.sh | KEPTN_VERSION=0.8.2 bash
```
Comment (christian-kreuzberger-dtx) [inline content/docs/0.8.x/operate/upgrade/index.md:25]: Same here, 0.8.3

File: content/docs/0.8.x/operate/upgrade/index.md (hunk 1 above)
Comment (johannes-b) [reply content/docs/0.8.x/operate/upgrade/index.md:23]: thx

File: content/docs/0.8.x/operate/upgrade/index.md (hunk 2 above)
Comment (johannes-b) [reply content/docs/0.8.x/operate/upgrade/index.md:25]: fixed it.

File: content/docs/0.8.x/operate/upgrade/index.md [hunk 3]
```diff
@@ -6,9 +6,92 @@ aliases:
...
+```
+    
+      * Verify that the installation has worked and that the version is correct by running:
+```console
+keptn version
+```
+
+    * **Manual installation of the Keptn CLI:**
+
+      * Download the release for your platform from the [GitHub](https://github.com/keptn/keptn/releases/tag/0.8.2)
```
Comment (christian-kreuzberger-dtx) [inline content/docs/0.8.x/operate/upgrade/index.md:35]: 0.8.3?

File: content/docs/0.8.x/operate/upgrade/index.md [hunk 4]
```diff
@@ -6,9 +6,92 @@ aliases:
...
+
+1. To upgrade your Keptn installation from 0.8.2 to 0.8.3, the Keptn CLI offers the command: 
+```console
+keptn upgrade
+```
+    
+    * Please [verify that you are connected to the correct Kubernetes cluster](../../troubleshooting/#verify-kubernetes-context-with-keptn-installation)
+before executing this command.
+    
+    * This CLI command executes a Helm upgrade using the Helm chart from: [keptn-installer/keptn-0.8.2.tgz](https://storage.googleapis.com/keptn-installer/keptn-0.8.2.tgz)
```
Comment (christian-kreuzberger-dtx) [inline content/docs/0.8.x/operate/upgrade/index.md:52]: 0.8.3?

File: content/docs/0.8.x/operate/upgrade/index.md (hunk 3 above)
Comment (johannes-b) [reply content/docs/0.8.x/operate/upgrade/index.md:35]: many thanks! - fixed it

File: content/docs/0.8.x/operate/upgrade/index.md (hunk 4 above)
Comment (johannes-b) [reply content/docs/0.8.x/operate/upgrade/index.md:52]: true!

**Review by laneli — APPROVED** [review]

File: content/docs/0.8.x/operate/upgrade/index.md [hunk 5]
```diff
@@ -6,9 +6,92 @@ aliases:
...
+**Note:** If you have manually modified your Keptn deployment, e.g., you deleted the Kubernetes Secret `bridge-credentials` for disabling basic auth, the `keptn upgrade` command will not detect the modification. Please re-apply your modification after performing the upgrade.
+
+### Step 2: Update your Shipyard for the Remediation Use-Case
+
+1. (optional) Set the Git-upstream repository to get access to your Shipyard as explained [here](../../manage/git_upstream/#create-keptn-project-or-set-git-upstream).
+
+1. Go to the Git repository and open the `shipyard.yaml` file for each project where you want to enable the remediation use-case.
+
+1. Copy-paste the following sequence to the stage, which should automatically remediate your problems:
+    *Remediation sequence*
```
[the remark this answers is not in the document]
Comment (johannes-b) [reply content/docs/0.8.x/operate/upgrade/index.md]: fixed it :)

**Pull request merged**